tests/components/webostv/test_config_flow.py

```diff
@@ -0,0 +1,364 @@
+"""Test the WebOS Tv config flow."""
+from unittest.mock import patch
+
+from aiopylgtv import PyLGTVPairException
+
+from homeassistant import config_entries
+from homeassistant.components import ssdp
+from homeassistant.components.webostv import CannotConnect
+from homeassistant.components.webostv.const import CONF_ON_ACTION, CONF_SOURCES, DOMAIN
+from homeassistant.config_entries import SOURCE_SSDP
+from homeassistant.const import (
+    CONF_CLIENT_SECRET,
+    CONF_HOST,
+    CONF_ICON,
+    CONF_NAME,
+    CONF_SOURCE,
+)
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from tests.common import MockConfigEntry
+
+MOCK_YAML_CONFIG = {
+    CONF_HOST: "1.2.3.4",
+    CONF_NAME: "fake",
+    CONF_ICON: "mdi:test",
+}
+
+MOCK_CONFIG_ENTRY = {
+    CONF_HOST: "1.2.3.4",
+    CONF_CLIENT_SECRET: "0123456789",
+}
+
+
+async def test_form_import(hass, client):
+    """Test we can import yaml config."""
+    with patch(
+        "homeassistant.components.webostv.config_flow.async_control_connect",
```

Comment:
We should be patching the PyPI and not the integration code `async_control_connect` https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations

Reply:
Tests are now patching only the `WebOsClient` from PyPI except for config flow tests which patch `async_setup_entry` to speed up tests by avoiding creating an entry